src/ResourceManager/RecoveryServices/Commands.RecoveryServices.Test/ScenarioTests/RecoveryServicesTests.ps1

```diff
@@ -52,4 +52,33 @@ function Test-RecoveryServicesVaultCRUDTests
 	Remove-AzureRmRecoveryServicesVault -Vault $vaultToBeProcessed
 	$vaults = Get-AzureRmRecoveryServicesVault -ResourceGroupName RsvTestRG -Name rsv1
 	Assert-True { $vaults.Count -eq 0 }
+}
+
+function Test-RecoveryServicesVaultCredFileDownloadTest
+{
+	# Create vault
+	$vaultCreationResponse = New-AzureRmRecoveryServicesVault -Name rsv1 -ResourceGroupName RsvTestRG -Location westus
+	Assert-NotNull($vaultCreationResponse.Name)
+	Assert-NotNull($vaultCreationResponse.ID)
+	Assert-NotNull($vaultCreationResponse.Type)
+
+	$drives = Get-PSDrive -PSProvider 'FileSystem'
+	$folderPath = $drives[0].Root
+	$file = Get-AzureRmRecoveryServicesVaultSettingsFile -Vault $vaultCreationResponse -Backup -Path $path
+	Assert-True { Test-Path $file.FilePath }
+	$fileContent = Get-Content $file.FilePath
+	Assert-ContainsWord($fileContent, $vaultCreationResponse.Name)
+	Assert-ContainsWord($fileContent, $vaultCreationResponse.ResourceGroupName)
+	Assert-ContainsWord($fileContent, $vaultCreationResponse.Location)
+
+	# Remove Vault
+	Remove-AzureRmRecoveryServicesVault -Vault $vaultCreationResponse
+	$vaults = Get-AzureRmRecoveryServicesVault -ResourceGroupName RsvTestRG -Name rsv1
+	Assert-True { $vaults.Count -eq 0 }
+}
+
+function Assert-ContainsWord($fileContent, $word)
```

Comment:
There is already an Assert-Contains function in the common library, what does this function do additionally?